lib/dovico/model/time_entry_formatter.rb

```diff
@@ -0,0 +1,41 @@
+module Dovico
+  class TimeEntryFormatter
+
+    def initialize(projects)
+      @projects = projects
+    end
+
+    def formatted_text(time_entries)
```

Comment:
why don't you override the `to_s` method? Do you need to be able to do a normal `to_s` on this model somewhere else?